### PR TITLE
replace chosen with typeahead for email notify

### PR DESCRIPTION
**Pull request by rh-kgarg**

### CODE REVIEW

**Comment by rh-kgarg** [comment]
@jeudy100 @vrathee @anujsi Please review.

File: app/cases/views/emailNotifySelect.jade [hunk 1]
```diff
@@ -11,15 +11,16 @@
...
-            ng-model='selectedUsers',
-            ng-change='selectedUsersChanged()',
-            width='"100%"',
-            ng-options="user.sso_username as (user.first_name+' '+user.last_name+' <'+user.sso_username+'>') for user in usersOnAccount | orderBy:['first_name','last_name']")
+            div.label-contact(ng-repeat="user in mapAllUsers(selectedUsers)", ng-class="{'label-disabled':saving}")
+                span {{user.first_name}} {{user.last_name}} <{{user.sso_username}}>
+                a(ng-click="removeUser(user.sso_username)", ng-disabled="saving")
+                    i.fa.fa-close
+            input.form-control.email-typeahead(type="text", placeholder="{{'Search here to add a User' | translate}}", ng-disabled="saving", 
+            typeahead-loading="CaseService.isLoadingRHUsers"
```
Comment (vrathee) [inline app/cases/views/emailNotifySelect.jade]: @rh-kgarg Can you please confirm its the right boolean flag for showing a loading state? I think its CaseService.usersLoading

Comment (rh-kgarg) [reply app/cases/views/emailNotifySelect.jade]: @vrathee yes..it should be usersLoading..fixing it next commit

Comment (rh-kgarg) [reply app/cases/views/emailNotifySelect.jade]: fixed it.

**Pull request merged**